packages/terra-open-source-scripts/src/terra-cli/prepare-for-release/prepareForReleaseMonoRepoHandler.js

```diff
@@ -0,0 +1,33 @@
+const childProcess = require('child_process');
+const fs = require('fs-extra');
+const path = require('path');
+const { promisify } = require('util');
+const stripAnsi = require('strip-ansi');
+const Logger = require('@cerner/terra-cli/lib/utils/Logger');
+
+const logger = new Logger({ prefix: '[terra-open-source-scripts:version]' });
+
+const exec = promisify(childProcess.exec);
+
+const updateChangelogForPackage = require('./updateChangelogForPackage');
+
+const VERSION_OUTPUT_PATH = path.join(process.cwd(), 'version-output.txt');
```

Comment:
```suggestion
const VERSION_OUTPUT_PATH = path.join(process.cwd(), 'tmp', 'version-output.txt');
```

Reply:
I'm fine doing this, but we delete the file so it shouldn't necessarily be hanging around or anything.

Reply:
https://github.com/cerner/terra-toolkit/pull/460/commits/6c35f4191a2115b9652c22dc9f42105554d0b08c